winlogbeat/config/config.go

```diff
@@ -36,7 +36,7 @@ type Settings struct {
 // Validate validates the Settings data and returns an error describing
 // all problems or nil if there are none.
 func (s Settings) Validate() error {
-	validKeys := []string{"winlogbeat", "output", "shipper", "logging"}
+	validKeys := []string{"winlogbeat", "output", "beat", "logging"}
```

Comment:
I think it will become a `ucfg.Option` that can be passed to `Unpack` that causes an error when a key is encountered that does not exist in the struct. I think urso has an open ticket in the repo to add the feature. 

Once it's implemented this can be removed. It would hard to move to libbeat because not all Beats put their options under a top-level "beatname" key.